tx-source/dash_core.php

```diff
@@ -718,6 +718,7 @@
 $Definition['The homepage was saved successfully.'] = 'The homepage was saved successfully.';
 $Definition['The invitation was removed successfully.'] = 'The invitation was removed successfully.';
 $Definition['The invitation was sent successfully.'] = 'The invitation was sent successfully.';
+$Definition['The Key in the XML payload to designate %s'] = 'The Key in the XML payload to designate %s';
```

Comment:
isn't this the key 
'The Key in the XML payload to designate Roles.'

Why did we need %s